.pipelines/submodules-pipeline.yaml

```
@@ -0,0 +1,300 @@
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    exclude:
+    - ".*"
+    - docs
+
+trigger:
+  paths:
+    include:
+    - "zapai/*"
+    - "azure-ipam/*"
+    - "dropgz/*"
+    exclude:
+    - docs
+  tags:
+    include:
+    - "zapai/*"
+    - "azure-ipam/*"
+    - "dropgz/*"
```

for includes to work as intended here, do we need to exclude everything else?
```suggestion
pr:
  branches:
    include:
      - master
  paths:
    exclude:
    - "*"
    include:
    - "zapai/*"
    - "azure-ipam/*"
    - "dropgz/*"

trigger:
  paths:
    exclude:
    - "*"
    include:
    - "zapai/*"
    - "azure-ipam/*"
    - "dropgz/*"
  tags:
    exclude:
    - "*"
    include:
    - "zapai/*"
    - "azure-ipam/*"
    - "dropgz/*"
```

---

yeah, this would make more sense